CITATION.cff

```diff
@@ -1,26 +1,43 @@
 cff-version: 1.2.0
-message: "If you cite ESBMC >= 5.0 please cite us as below."
+message: "If you cite ESBMC >= 7.0 please cite us as below."
 authors:
-    - family-names: Gadelha
-      given-names: Mikhail R.
+    - family-names: "Sá Menezes"
+      given-names: Rafael
+      orcid: https://orcid.org/0000-0002-6102-4343
+    - family-names: Aldughaim
+      given-names: Mohannad
+      orcid: https://orcid.org/0000-0003-1708-1399
+    - family-names: Farias
+      given-names: Bruno
+    - family-names: Xianzhiyu
+      given-names: Li
```

Comment:
```suggestion
      given-names: Xianzhiyu
```

Reply:
I'm very sorry, I should have know better. Thanks.